packages/pwa-kit-cli/bin/cli.js

```diff
@@ -126,6 +126,16 @@ const main = () => {
                     ...process.env
                 }
             })
+
+            // Copy the project `package.json` into the build folder.
+            sh.cp(p.resolve('package.json'), './build/package.json')
+
+            // Copy config files.
+            const config = p.resolve('config')
+            if (fs.existsSync(config)) {
+                sh.cp('-R', config, './build')
```

Comment:
I saw that we ignored copying local.* previously. Do we want to do the same here?

Reply:
Good catch. I'll look into that